File: parantier-front/src/entities/work/types/workMindmap.ts
export interface WorkMindmap {
  id: number
  workId: number
  title: string
  content: string
  orderNum: number
  createdAt: string
  updatedAt: string
}

export interface CreateWorkMindmapRequest {
  title: string
  content: string
  orderNum?: number
}

export interface UpdateWorkMindmapRequest {
  title: string
  content: string
  orderNum: number
}

File: parantier-api/src/main/java/com/mapo/palantier/work/domain/WorkMindmap.java
package com.mapo.palantier.work.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class WorkMindmap {
    private Long id;
    private Long workId;
    private String title;
    private String content;
    private Integer orderNum;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}
